File: cms_server/cms_server/Entities/ConsultMedEntity.cs
using System;

namespace cms_server.Entities
{
    public class ConsultMedEntity
    {
        public int? cr_med_pk { get; set; }
        public string consult_req_pk { get; set; }
        public string med_no { get; set; }
        public string med_desc { get; set; }
        public string unit { get; set; }
        public string dosage { get; set; }
        public string is_active { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }
    }
}

File: cms_web/src/Services/Entities/ConsultMedEntity.ts
interface ConsultMedEntity {
  cr_med_pk?: number;
  consult_req_pk?: string;
  med_no?: string;
  med_desc?: string;
  unit?: string;
  dosage?: string;
  is_active?: string;
  encoded_at?: string | Date;
  encoder_pk?: string;

  //
  attach_file?: string;
}

export interface ConsultMedTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<ConsultMedEntity>;
}

export default ConsultMedEntity;
